torch/_refs/__init__.py

```diff
@@ -923,6 +923,30 @@ def div(
     supports_lhs_python_scalar=False,
 )
 
+
+def _pow(
+    a: Union[TensorLikeType, NumberType],
+    b: Union[TensorLikeType, NumberType],
+) -> TensorLikeType:
+    assert isinstance(a, TensorLikeType) or isinstance(b, TensorLikeType)
+
+    if isinstance(b, Number):
+        if b == 1.0:
+            return a  # type: ignore[return-value]
```

Comment:
It seems dangerous to sometimes return a new tensor and sometimes not
```suggestion
            return a.clone()  # type: ignore[return-value]
```

Reply:
Good catch, @peterbell10! `torch.pow` always returns a new tensor